01_prepare_host.sh

```diff
@@ -13,19 +13,7 @@ else
   # shellcheck disable=SC1091
   source centos_install_requirements.sh
 fi
-
-if ! which minikube 2>/dev/null ; then
```

Comment:
The access rights on this file have changed!

Reply:
That's why the CI failed

Reply:
What is the original access rights on this file ?

Reply:
Fixed. Please trigger the test